### PR TITLE
Fixing regression where RD2 Change Type is not set without value change

### PR DESCRIPTION
**Pull request by lparrott**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/lwc/rd2Service/model.js [hunk 1]
```diff
@@ -134,7 +135,7 @@ const getDonationValue = (state) => {
 
 const getChangeType = (state) => {
     if (isRecurringTypeChanged(state) || isNewRecord(state)) {
-        return "";
+        return state.changeType;
```
Comment (lparrott) [inline force-app/main/default/lwc/rd2Service/model.js:138]: Doing this allows a manual change to the picklist to "stick" even if the overall Value changed, which is in line with our current release.

Comment (lparrott) [reply force-app/main/default/lwc/rd2Service/model.js:138]: Yup, I thought about this last night after you reached out, I'll make the change and do a bit more testing!

Comment (lparrott) [reply force-app/main/default/lwc/rd2Service/model.js]: Just pushed a fix, thanks for adding that Jest test 👍

**Review by bdvorachek — APPROVED** [review]

**Review by andrewyu-salesforce — APPROVED** [review]

**Comment by lparrott** [comment]
Added to #6928

**Pull request closed without merging**